lib/skate_web/views/layout_view.ex

```diff
@@ -1,5 +1,6 @@
 defmodule SkateWeb.LayoutView do
-  use SkateWeb, :view
+  use(SkateWeb, :html)
+  use(SkateWeb, :view)
```

Comment:
nitpick: In the past the formatter hasn't forced parentheses on `use` statements, do you know why it's happened here / what changed?

Reply:
I have absolutely no idea. I assumed it _was_ a formatter thing, which is why I didn't change it back, but I just fixed it and the formatter isn't fighting me.